### PR TITLE
[18.0][MIG] connector_importer: Migration to 18.0

### PR DESCRIPTION
**Pull request by thienvh332**

### CODE REVIEW

File: connector_importer/models/sources/source_mixin.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        """Collect data for summary."""
+        return {
+            "source": self,
+            "summary_fields": self._config_summary_fields,
+            "fields_info": self.fields_get(self._config_summary_fields),
+        }
+
+    # TODO: check if still needed + use create multi
+    @api.model
+    def create(self, vals):
```
Comment (nguyenminhchien) [inline connector_importer/models/sources/source_mixin.py]: DeprecationWarning: The model odoo.addons.connector_importer.models.sources.source_mixin is not overriding the create method in batch

Comment (nguyenminhchien) [reply connector_importer/models/sources/source_mixin.py]: Check and implement if possible for     # TODO: check if still needed + use create multi

File: connector_importer/views/backend_views.xml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                            You will not be able to run the import again or to delete this backend
+                            until you complete the jobs.
+                    </div>
+                    <notebook>
+                        <page string="Recordsets">
+                            <!-- TODO: Create active_id field if you want to use default_backend_id. -->
+                            <field
+                                name="recordset_ids"
+                                nolabel="1"
+                                context="{'default_backend_id': False}"
```
Comment (nguyenminhchien) [inline connector_importer/views/backend_views.xml]:
```suggestion
                                context="{'default_backend_id': id}"
```

File: connector_importer/views/backend_views.xml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                        invisible="job_running == False"
+                    >
+                        <strong>JOBS RUNNING</strong>
+                            WARNING: one or more jobs are scheduled for a recorset or a record.
+                            You will not be able to run the import again or to delete this backend
+                            until you complete the jobs.
+                    </div>
+                    <notebook>
+                        <page string="Recordsets">
+                            <!-- TODO: Create active_id field if you want to use default_backend_id. -->
```
Comment (nguyenminhchien) [inline connector_importer/views/backend_views.xml]:
```suggestion
                          
```

File: connector_importer/readme/CREDITS.md [hunk 4]
```diff
@@ -0,0 +1 @@
+The migration of this module from 16.0 to 18.0 was financially supported by Camptocamp.
```
Comment (nguyenminhchien) [inline connector_importer/readme/CREDITS.md]: From the changes in __manifest__.py, i saw the version changed from 17.0 to 18.0

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]
Add `odoo_test_helper` to tests-requirement.txt

File: connector_importer/readme/CREDITS.md (hunk 4 above)
Comment (thienvh332) [reply connector_importer/readme/CREDITS.md]: I started from v16 and I only guaranteed v18. So I wrote v16 --> v18. (v17 I just put it up to get their automatic migration script =)))

**Review by nguyenminhchien — APPROVED** [review]

**Pull request closed without merging**